lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -761,6 +702,35 @@ impl Generator {
         let l2_precomputed = PrerenderedU8Function::new(metric_l2);
         let mut total_processed_pixels = 0;
         let max_workers = params.max_thread_count;
+        let mut has_fanned_out = false;
+
+        {
+            // now that we have all of the parameters we can setup our initial tree grid
+            let tile_adjusted_width = (self.output_size.width as f32 * 1.1) as u32 + 1;
+            let tile_adjusted_height = (self.output_size.height as f32 * 1.1) as u32 + 1;
+            self.tree_grid = TreeGrid::new(
+                tile_adjusted_width,
+                tile_adjusted_height,
+                max(tile_adjusted_width, tile_adjusted_height),
+                (self.output_size.width as f32 * 0.05) as u32 + 1,
+                (self.output_size.height as f32 * 0.05) as u32 + 1,
```

Comment:
What's the reasoning behind the `1.1` multiplier, and why is `1` added on top of it?

Reply:
This was just a low quality thing on my part. The 1.1 is because in tiling mode you have boundaries which are 5% of the image size. I will move that into a constant. You are also right about the +1 being for rounding up. I will follow your suggestion below.